indico/modules/events/papers/templates/management/assignment.html

```diff
@@ -0,0 +1,95 @@
+{% extends 'layout/full_content_base.html' %}
+{% from 'events/papers/management/_paper_assignment_list.html' import render_paper_assignment_list %}
+{% from 'events/management/_lists.html' import render_displayed_entries_fragment %}
+
+{% block title %}
+    {% trans %}Paper Assignment{% endtrans %}
+{% endblock %}
+
+{% block description %}
+    {%- trans -%}
+        Assign <strong>judges</strong>, <strong>content reviewers</strong> and <strong>layout reviewers</strong>
```

Comment:
*judges and reviewers* to avoid having to use conditionals depending on what's enabled (for such conditionals you'd need to repeat the whole sentence since you cannot have them inside trans blocks)